client/my-sites/site-settings/site-settings-form-fieldset/index.jsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import FormFieldset from 'components/forms/form-fieldset';
+import InfoPopover from 'components/info-popover';
+
+export const SiteSettingsFormFieldset = ( { isJetpackSite, children, className } ) => {
```

Comment:
Do we really need this component? If we do, we might want to rename it, as it appears to be specific to Jetpack site settings.